ci-operator/config/shiftstack/merge-bot/shiftstack-merge-bot-main__periodic-4.12.yaml

```diff
@@ -0,0 +1,119 @@
+build_root:
+  project_image:
+    dockerfile_literal: |-
+      FROM registry.access.redhat.com/ubi8/python-39
+
+      # Prow expects to be able to check out a repo under /go
+      USER 0:0
+      WORKDIR /go
+      RUN chown 1001:1001 .
+
+      USER 1001:1001
+images:
+- dockerfile_path: Containerfile
+  to: merge-bot
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: make install-all-deps
+tests:
+- as: lint
+  commands: make lint
+  container:
+    from: test-bin
+- as: unittests
+  commands: make unittests
+  container:
+    from: test-bin
+- as: cloud-provider-openstack
+  cron: 0 12 * * Mon,Thu
+  steps:
+    test:
+    - as: cloud-provider-openstack
+      commands: |
+        merge-bot --source https://github.com/kubernetes/cloud-provider-openstack:release-1.25 \
+                  --dest openshift/cloud-provider-openstack:release-4.12 \
+                  --merge shiftstack/cloud-provider-openstack:merge-bot-release-4.12 \
+                  --update-go-modules \
+                  --bot-name shiftstack-merge-bot --bot-email shiftstack-merge-bot@redhat.com \
+                  --github-app-key /secrets/merge-bot/github_private_key \
+                  --github-cloner-key /secrets/merge-bot/github_cloner_private_key \
+                  --slack-webhook /secrets/slack-hooks/forum-shiftstack
+      credentials:
+      - mount_path: /secrets/merge-bot
+        name: shiftstack-merge-bot
+        namespace: test-credentials
+      - mount_path: /secrets/slack-hooks
+        name: shiftstack-slack-hooks
+        namespace: test-credentials
+      from: merge-bot
+      resources:
+        limits:
+          memory: 4Gi
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: csi-driver-nfs
```

Comment:
Hmmm, csi-driver-nfs doesn't follow the same release schedule as the rest of kubernetes it seems: https://github.com/kubernetes-csi/csi-driver-nfs/branches